File: 7-kyu/complementary-dna/complementary-dna.js
function dnaStrand(dna){
   const complements = {
    A: "T",
    T: "A",
    C: "G",
    G: "C"
  };
  
  return dna
    .split("") // Split the string into individual characters
    .map(base => complements[base]) // Map each base to its complement
    .join(""); // Join the complementary bases into a string
}

File: 7-kyu/complementary-dna/complementary-dna.py
def DNA_strand(dna):
    str = []
    for c in dna:
        if c == "A":
            str.append("T")
        elif c == "T":
            str.append("A")
        elif c == "C":
            str.append("G")
        elif c == "G":
            str.append("C")
    return "".join(str)